.github/workflows/PR.yaml

```diff
@@ -0,0 +1,101 @@
+              - '**.py'
+              - 'sovrin/*genesis'
+  extract-information:
+    name: Extract needed Information for Indy-Test-Automation
+    needs: [workflow-setup]
+    if: ${{ needs.workflow-setup.outputs.testsNeeded == 'true' }}
+    runs-on: ubuntu-latest
+    outputs:
+      indyVersion: ${{ steps.indy-versions.outputs.indyVersion}}
+      sovtokenVersion: ${{ steps.sovtoken-versions.outputs.sovtokenVersion}}
+      sovtokenfeesVersion: ${{ steps.sovtoken-versions.outputs.sovtokenfeesVersion}}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.8'
+      - name: Set up python
+        run: |
+          pip3 install semver    
+      - name: Get Version with UX-timestamp
+        id: version
+        run: |
+          echo "version=$(python3 updateVersion.py --getVersion)" >> $GITHUB_OUTPUT
+      - name: prepare
+        run: ./build-scripts/ubuntu-1604/prepare-package.sh ./ ${{ steps.version.outputs.version }}
+      - name: Extract Indy Versions
+        id: indy-versions
+        run:  |
+          echo "indyVersion=$(grep -oP "\d+.\d+.\d+((-|.)?rc\d+)?" <<< $(grep -oP "indy-node==\d+.\d+.\d+((-|.)?rc\d+)?" setup.py))" >>$GITHUB_OUTPUT
+      - name: Extract sovtoken & sovtokenfees Version
+        id: sovtoken-versions
+        run: |
+          echo "sovtokenVersion=$(grep -oP "\d+.\d+.\d+((-|.)?((rc)|(dev))?\d+)?" <<< $(grep -oP "sovtoken==\d+.\d+.\d+((-|.)?(rc|dev)\d+)?" setup.py))" >>$GITHUB_OUTPUT
+          echo "sovtokenfeesVersion=$(grep -oP "\d+.\d+.\d+((-|.)?((rc)|(dev))?\d+)?" <<< $(grep -oP "sovtokenfees==\d+.\d+.\d+((-|.)?(rc|dev)\d+)?" setup.py))" >>$GITHUB_OUTPUT
+  
+  build_sovrin:
+    name: Build Sovrin Deb
+    needs: [workflow-setup]
+    if: ${{ needs.workflow-setup.outputs.testsNeeded == 'true' }}
+    uses: ./.github/workflows/reuseable_build_package.yaml
+    with:
+      isDev: true    
+
+  call-indy-test-automation:
+    name: Indy-Test-Automation
+    needs: [workflow-setup, extract-information, build_sovrin]
+    if: ${{ needs.workflow-setup.outputs.testsNeeded == 'true' }}    
+    uses: pschlarb/indy-test-automation/.github/workflows/callable_test-automation.yaml@sovrin
+    with:
+      ubuntuVersion: "ubuntu-2004"
+      libsovtokenJSON: "{\"install\": \"yes\",  \"version\": \"1.0.5\"}"
+      tokenPluginJSON: "{\"install\":\"yes\", \"sovtokenVersion\": \"${{ needs.extract-information.outputs.sovtokenVersion }}\", \"sovtokenfeesVersion\":\"${{ needs.extract-information.outputs.sovtokenfeesVersion }}\"}"
+      sovrinJSON: "{ \"install\": \"yes\"}"
+      indyJSON: "{ \"nodeVersion\": \"${{ needs.extract-information.outputs.indyVersion }}\", \"plenumVersion\" : \"1.13.1~rc2\", \"ursaVersion\": \"0.3.2-1\", \"pyzmqVersion\" : \"22.3.0\"}"
+      libIndyVersion: "1.15.0~1625-bionic"
+      includePaymentTests: true
+      testAutomationBranch: "sovrin"
```

Comment:
Needs to be changed to the correct branch after https://github.com/hyperledger/indy-test-automation/pull/120/ is merged